xbmc/cores/DllLoader/exports/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ elseif(NOT CORE_SYSTEM_NAME STREQUAL windows)
   add_custom_target(wrapper.def ALL nm ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/wrapper.dir/wrapper.c.o | grep __wrap | awk '{ printf(\"%s \", \$\$3) }' | sed \"s/___wrap_/__wrap_/g\" | sed \"s/__wrap_/-Wl,-wrap,/g\" > wrapper.def)
 
   if(CORE_SYSTEM_NAME STREQUAL android)
-    add_custom_command(TARGET wrapper.def COMMAND echo \"-l$<TARGET_FILE:${APP_NAME_LC}>\" >> wrapper.def)
+    add_custom_command(TARGET wrapper.def COMMAND echo \"-L${kodi_BINARY_DIR} -l:libkodi.so\" >> wrapper.def)
     add_dependencies(wrapper.def ${APP_NAME_LC})
```

Comment:
its fom an android documentation with ":", seems to work well.
But, why use dummy? The problem seems more that the full path is given in -l